Sources/RTMP/RTMPStream.swift

```diff
@@ -371,6 +371,9 @@ open class RTMPStream: IOStream {
             guard let connection = self.connection, self.readyState == .publishing(muxer: self.muxer) else {
                 return
             }
+            if handlerName == "@setDataFrame" {
+                self.dataTimeStamps.removeValue(forKey: handlerName)
+            }
```

Comment:
When considering Wowza, it becomes necessary to send timestamp differential. This will be marked as a change request as it could lead to a bug in Wowza.

Reply:
When there is no value, the chunk type will be 0 which expects the absolute timestamp instead of delta. But looks like sending new absolute timestamp requires resetting some other values in the code. I will test Wowza next week. Maybe there could be a new argument in `send` specifying chunk type, it would also reset all timestamps.

Reply:
Since the specification for @setDataFrame is undefined, it relies on server-side implementation. For the target Wowza, it was necessary to send it in this manner. However, it seems to not function correctly with nginx. If supporting Twitch behavior, providing an isResetTimestamp option seems like a good approach. https://github.com/shogo4405/HaishinKit.swift/pull/1390
https://github.com/shogo4405/HaishinKit.swift/blob/0a48aee6072f805537389e235b449934d06c7a34/Sources/RTMP/RTMPStream.swift#L368-L379

Reply:
Yes this works well. I tested YouTube, Twitch and Wowza with `isResetTimestamp: true`. Can't verify nginx. Closing this PR.